celery_serverless/handler.py

```diff
@@ -62,6 +62,12 @@ def worker(event, context):
     global hooks
 
     request_id = '(unknown)'
+    if 'wdb' in available_extras:
+        available_extras['wdb']['start_trace']()
+        if available_extras['wdb']['breakpoint']:
+            import wdb
+            wdb.set_trace()
```

Comment:
Maybe add some comments so that when the user debugger stops here, he can understand where his task code is executed.